extensions-core/protobuf-extensions/src/main/java/org/apache/druid/data/input/protobuf/FileBasedProtobufBytesDecoder.java

```diff
@@ -98,35 +98,40 @@ public DynamicMessage parse(ByteBuffer bytes)
         url = new URL(descriptorFilePath);
       }
       catch (MalformedURLException e) {
-        throw new ParseException(e, "Descriptor not found in class path or malformed URL:" + descriptorFilePath);
+        throw new ParseException(
+            descriptorFilePath,
```

Comment:
nit: descriptorFilePath (and some of the other things passed into the input argument in various places) don't really seem to be stringified input, should these that don't have the actual row be using `null` here instead?

Reply:
I made some of the non-input ones null